src/bundler.ts

```diff
@@ -90,8 +90,10 @@ export class Bundler {
 
     this.excludes = Array.isArray(opts.excludes) ? opts.excludes : [];
     this.stripComments = Boolean(opts.stripComments);
-    this.enableCssInlining = Boolean(opts.inlineCss);
-    this.enableScriptInlining = Boolean(opts.inlineScripts);
+    this.enableCssInlining =
+        Boolean(typeof opts.inlineCss === 'undefined' || opts.inlineCss);
```

Comment:
we can just do `opts.inlineCss === undefined`. We're not going to redefine undefined in any of our code. Setting `undefined` on the global does nothing, so we'd have to set it in one of our modules for this to be a problem.